cibersort/macros.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<macros>
+
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="3.6.2">R</requirement>
+            <requirement type="package">reshape2</requirement>
+            <requirement type="package">ggplot2</requirement>
+            <requirement type="package">dplyr</requirement>
+            <requirement type="package">e1071</requirement>
+            <requirement type="package">parallel</requirement>
+            <requirement type="package">preprocess</requirement>
+            <requirement type="package" version="1.04">CIBERSORT</requirement>
+        </requirements>
```

Comment:
Check package names; these won't resolve with conda. Check dplyr is still needed. Remove CIBERSORT (conda package does not exist).

Reply:
`preprocess` -> `preprocessCore`, `CIBERSORT` removed, `parallel` should be correct.